README.md

```diff
@@ -8,6 +8,33 @@ Demo: https://www.youtube.com/watch?v=HXdQd65CwkU
 ## Status
 WIP but playable.
 
+## Installation
+
+Prebuilt binary releases are available if you're looking for testing the ruleset or don't possess a developpment environment:
+| [Releases Page](https://github.com/swoolcock/rush/releases/) |
+| -------------|
+
+**NOTE**: For the moment, only desktop platforms (Windows, mac OSX, Linux) are supported.
```

Comment:
```suggestion
**NOTE**: osu!lazer runs custom rulesets only on desktop platforms (Windows, macOS, Linux) for the time being.
```
Better provide **what** is not supporting to avoid confusion.